src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -183,9 +199,58 @@ class PolicyHandlerTest : public ::testing::Test {
     EXPECT_CALL(*mock_policy_manager_, TimeoutExchange()).WillOnce(Return(1));
     EXPECT_CALL(*mock_policy_manager_, RetrySequenceDelaysSeconds())
         .WillOnce(Return(retry_sequence_delay_seconds));
-    EXPECT_CALL(*MockMessageHelper::message_helper_mock(),
-                SendPolicyUpdate(_, _, _, _));
+    EXPECT_CALL(mock_message_helper_, SendPolicyUpdate(_, _, _, _));
   }
+
+  void OnPermissionsUpdated(const std::string& default_hmi_level,
```

Comment:
@okozlovlux rename it to TestOnPermissionsUpdated() 
Because it is not clear from tests that you have some expects there, not onlly logik